integrationExamples/gpt/pbjs_example_gpt.html

```diff
@@ -582,6 +359,20 @@ <h2>Prebid.js Test</h2>
     });
 </script>
 
+<script>
+    var postIASResponse = function(res) {
+    	var div = document.getElementsByClassName('fw-response')[0];
+    	if (div) {
+    		div.textContent = JSON.stringify(res);
+        }
+    }
+</script>
+
+<h4>FW response</h4>
+<div class="fw-response"></div>
+<h4>DFP keywords</h4>
+<div class="dfp-keywords"></div>
+
```

Comment:
We also probably shouldn't be committing change to this test page to Prebid/Prebid.js - but we can keep it in our forked repo.